raiden/tests/utils/network.py

```diff
@@ -313,6 +313,7 @@ def create_apps(
                             'retry_interval': retry_interval,
                             'server': local_matrix_url,
                             'server_name': 'matrix.local.raiden',
+                            'private_rooms': True,
```

Comment:
Yes, I think so, but I don't want to duplicate all test time only because of a dual behavior which should expose the same interface (outside transport, nothing changes).. will try to parametrize a single test for it